capeinfra/iam.py

```diff
@@ -337,12 +337,13 @@ def get_inline_role(
             opts=opts,
         )
 
-    # and now add the policy rules we were given to the role
-    aws.iam.RolePolicy(
-        f"{name}-{srvc_prfx}roleplcy",
-        role=inline_role.id,
-        policy=role_policy,
-        opts=opts,
-    )
+    # and now add the policy rules we were given to the role if configured
+    if role_policy:
```

Comment:
```suggestion
    if role_policy is not None:
```

This is a great change, I actually have this in my other branch I'm working on (good minds think alike)

Reply:
This was pure oversight and your change is the def correct one.